openapi.yaml

```diff
@@ -0,0 +1,3036 @@
+openapi: 3.0.2
+# ImageViewDataSize is the total number of bytes in ImageViewData.ImageData.  Use of this field is NOT recommended, so it has not been included in the openapi.yaml
+info:
+  description: Moov X9 () implements an HTTP API for creating, parsing and validating X9 (Check21) files.
+  version: v1
+  title: X9 API
+  contact:
+    url: https://github.com/moov-io/x9
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: http://localhost:8083
+    description: Local development
+
+tags:
+  - name: Files
```

Comment:
Can we call this tag Wire? `Files` is already taken by ACH files in our [api docs](https://api.moov.io/#tag/Files).

Reply:
Adam and I had a slack chat, we are going to go with ACH Files, X9 Files, and Wire Files